docs/content/en/docs/install/install.md

```diff
@@ -102,30 +126,47 @@ modify some values, and use the modified file to install Keptn:
 
 1. Edit your local copy to modify some values
 
-1. Install Keptn by adding the following string to your `helm upgrade` command line:
+1. To apply the modified configuration.
+   install Keptn by adding the following string
+   to your `helm upgrade` command line:
 
    ```shell
    --values=values.yaml
    ```
 
-You can also use the `--set` flag
-to specify a value change for the `helm upgrade --install` command.
-Configuration options are specified using the format:
+   For example, if you create a `my.values.yaml`
+   and modified some configuration values,
+   the command sequence to apply your configuration is:
 
-```shell
---set key1=value1 \
---set key2=value2 ...
-```
+   ```shell
+   helm repo add keptn https://charts.lifecycle.keptn.sh
+   helm repo update
+   helm upgrade --install keptn keptn/keptn \
+    --values my.values.yaml \
+    -n keptn-lifecycle-toolkit-system --create-namespace --wait
+   ```
```

Comment:
I don't think we need this example here. We already say above that you just need to add the one flag to your already existing install command from above

Reply:
I don't feel strongly about this but @bacherfl thought it was a good idea and I do think that, for a lot of people, seeing the actual command (and being able to copy/paste it) enhances usability.  But if you and Flo agree, I'm good.

Reply:
same here. example is fine, but i think we don't need to repeat repo add and repo update

Reply:
Done.